web/web/api/v1/pv.py

```diff
@@ -151,4 +151,4 @@ def add_pv():
 
     result = PvSchema().dump(new_pv)
 
-    return arw.to_json(result)
+    return arw.to_json(result)
```

Comment:
Is there a particular reason you are removing new lines from the end of files? They should really be there....

Reply:
The line is there at the end of the file (return arw.to_json(result). Maybe it's appearing to have been changed/removed because of deleting spacing or formatting? This is the line you are referring to, right? Not exactly sure what new lines are being removed

Reply:
<img width="712" alt="Screen Shot 2020-07-01 at 2 31 22 PM" src="https://user-images.githubusercontent.com/5042601/86293499-8ac01c80-bba7-11ea-8e06-fcb5c3e7f560.png">
This is an example of what github is showing.